### DISCUSSION TITLE
How to format XML files using the HTML formatter

### DISCUSSION THREAD

**Discussion by jylin** [prettier/prettier#13473] [Q&A]
Is there a way to configure prettier to format XML files using the HTML formatter (I would even suggest adding *.xml to the extensions list for the HTML formatter)? I'd like to force this in my global prettier configuration.

The XML plugin (https://github.com/prettier/plugin-xml) has serious problems and basically does not work for many cases. The HTML formatter does a much better job.

**Comment by lulunac27a** [comment]
It's working when I followed the steps in [Getting Started](https://github.com/prettier/plugin-xml#getting-started) section in [prettier/plugin-xml](https://github.com/prettier/plugin-xml) repository. It also works with empty tag with custom tag name that is not supported in HTML like `<emptyTag/>`.

**Reply by CodeIt-Bro** [reply]
You can use this [online XML formatter](https://www.codeitbro.com/tool/xml-formatter).
